src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginDiscoverer.cs

```diff
@@ -63,6 +63,17 @@ internal class TestPluginDiscoverer
 
 #if !WINDOWS_UAP
 
+            if(!extensionPaths.Any())
+            {
+                // C++ UWP adaters do not follow TestAdapater naming convention, so making this exception)
+                extensionPaths = extensionPaths.Concat<string>(new List<string> { "Microsoft.VisualStudio.TestTools.CppUnitTestFramework.ComInterfaces.dll",
```

Comment:
Shouldn't this be an const array? Should there be a check if these files exist before adding them for discovery?

Reply:
adding check is not necessary, since it will fail at worst during assembly load, with file not file exception. I'm okay with it, since it doesn't break anything, and assembly load will check for file existence anyway